addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -93,6 +93,27 @@
 		<state pattern="%.2f A" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="channel1Amps" advanced="true">
+		<item-type>Number</item-type>
+		<label>Channel 1 Amps</label>
+		<description>Channel 1 current in Amperes</description>
+		<state pattern="%.2f A" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="channel2Amps" advanced="true">
```

Comment:
it imho does not make much sense to define 3 channel types - they are all the same type after all. Why not just introduce one "amps" channel type? Note that you can customise the label+description on the channel definitions in the "current" thing type itself.